manager/manager.go

```diff
@@ -225,6 +227,7 @@ func New(config *Config) (*Manager, error) {
 		listeners:   listeners,
 		caserver:    ca.NewServer(raftNode.MemoryStore(), config.SecurityConfig),
 		dispatcher:  dispatcher.New(raftNode, dispatcherConfig),
+		logbroker:   logbroker.New(),
```

Comment:
Should `logbroker` be started when we become the leader, and stop when we lose leadership? I'd think we want to close log streams when we lose leadership, and refuse to open new ones.

What happens right now when there's a leader failover? Do the agents keep publishing logs to the old leader as long as existing streams are open, or does the raft proxy direct these logs to the new leader instead? Will the client be able to automatically reestablish a stream on leader failover?